types/react-select/tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-    "extends": "dtslint/dt.json"
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-null-undefined-union": false
```

Comment:
Why's this necessary?

Reply:
dtslint use the `tslint:all` with a recent breaking change